### ISSUE TITLE
Inclusion in official osmocom rtlsdr codebase

### ISSUE THREAD

**Issue by jsr38** [gat3way/r820tweak#2]
Hi! 

I'm about to try this out.  It looks really useful.  Nice job.

Has this functionality been merged with the main rtl-sdr codebase? 

I'm looking through the logs and mailing list and there are discussions about things that sound similar.  Wonder if you could clarify.

**Comment by jsr38** [comment]
Hah! OK I think I see what's going on from the mailing list exchange...

**Comment by gat3way** [comment]
I don't think it is. I don't think it will ever be (at least not in that form).

**Comment by eabase** [comment]
Any updates?
